### DISCUSSION TITLE
library Universal Robot UR3e

### DISCUSSION THREAD

**Discussion by PavelGreen665** [visose/Robots#195] [Q&A] [answered]
Hi, is the xml and 3dm files available for Universal Robot UR3e please?

**Comment by madelinegannon** [comment, marked as answer]
There is now a `UniversalRobots` online library with every UR model, including UR3e.

**Reply by PavelGreen665** [reply]
including XML file?

**Reply by madelinegannon** [reply]
Yes 🎉 #209 
You can update the plugin from the Rhino PackageManager.
